Yosemite/YosemiteTests/Stores/ProductReview/RetrieveProductReviewFromNoteUseCaseTests.swift

```diff
@@ -14,27 +14,33 @@ final class RetrieveProductReviewFromNoteUseCaseTests: XCTestCase {
     private var notificationsRemote: MockNotificationsRemote!
     private var productReviewsRemote: MockProductReviewsRemote!
     private var productsRemote: MockProductsRemote!
+    private var storageManager: MockupStorageManager!
+
+    private var viewStorage: StorageType {
+        storageManager.viewStorage
+    }
 
     override func setUp() {
         super.setUp()
 
         notificationsRemote = MockNotificationsRemote()
         productReviewsRemote = MockProductReviewsRemote()
         productsRemote = MockProductsRemote()
+        storageManager = MockupStorageManager()
     }
 
     override func tearDown() {
+        storageManager = nil
         productsRemote = nil
         productReviewsRemote = nil
         notificationsRemote = nil
 
         super.tearDown()
     }
 
-    func testItFetchesAllEntitiesAndReturnsTheParcel() throws {
+    func test_it_fetches_all_entities_and_returns_the_Parcel() throws {
```

Comment:
nit: is `parcel` capitalized intentionally?

Reply:
I think so. 🙈 I think I wanted it to refer to [`ProductReviewFromNoteParcel`](https://github.com/woocommerce/woocommerce-ios/blob/061c2bce126fa84064d2125c45339be65269ab7a/Yosemite/Yosemite/Stores/ProductReview/RetrieveProductReviewFromNoteUseCase.swift#L8).